sui/wormhole/sources/state.move

```diff
@@ -86,7 +86,7 @@ module wormhole::state {
             governance_contract: external_address::from_bytes(governance_contract),
             guardian_set_index: u32::from_u64(0),
             guardian_sets: vec_map::empty<U32, GuardianSet>(),
-            guardian_set_expiry: u32::from_u64(0),
+            guardian_set_expiry: u32::from_u64(2), // TODO - what is the right #epochs to set this to?
```

Comment:
we should aim for at least 24h, it's not immediately obvious reading on sui's website what the epoch lengths are, they're seemingly decided at genesis so I guess technically we don't know yet until mainnet launches. Setting it to 2 for now makes sense

Reply:
I don't think the epoch's are updating right now anyhow.  At least last time I looked the RPC's were always returning 0

Reply:
at least the increment_epoch function works